apps/prairielearn/assets/scripts/instructorQuestionSettingsClient.ts

```diff
@@ -40,6 +41,27 @@ onDocumentReady(() => {
     },
   });
 
+  new TomSelect('#tags', {
+    valueField: 'name',
+    searchField: ['name', 'description'],
+    plugins: ['dropdown_input', 'remove_button'],
+    render: {
+      option(data: Tag) {
+        return html`
+          <div>
+            ${TagBadge(data)}
+            <div>
+              <small class="text-muted">${data.description}</small>
+            </div>
+          </div>
+        `.toString();
+      },
+      item(data: Tag) {
+        return html`<span class="badge color-${data.color} m-1">${data.name}</span>`.toString();
```

Comment:
The badge styles hide the default styles that `tom-select` adds to indicate interactivity. For example, go to https://tom-select.js.org/examples/ and interact with the first example. Mousing over a selected item shows a cursor to indicate it can be clicked; clicking selects it; one can Cmd/Shift-click to select multiple items and then remove them with backspace/delete. That mode of operation isn't clear without those styles.

Adding the following to the `<style>` tag on this page made things work better for me:

```css
          .ts-wrapper.multi .ts-control > span {
            cursor: pointer;
          }

          .ts-wrapper.multi .ts-control > span.active {
            /* Fallback for Bootstrap 4; we can remove when we drop support for it. */
            background-color: var(--bs-primary, #0d6efd) !important;
            color: white !important;
          }
```

Reply:
Added the styles above in [d426d26](https://github.com/PrairieLearn/PrairieLearn/pull/10973/commits/d426d26b01edba612257c99691275d279950c468).